setup.py

```diff
@@ -11,20 +11,18 @@
 with open('HISTORY.rst') as history_file:
     history = history_file.read()
 
+with open('requirements_test.txt') as test_requirements_file:
```

Comment:
Here I'm opeining the requirements file so that we can avoid redundant dependency lists